backend/typescript/models/scheduling.model.ts

```diff
@@ -0,0 +1,35 @@
+import { Column, DataType, Model, Table } from "sequelize-typescript";
+import { Status } from "../types";
+
+@Table({ tableName: "scheduling" })
+export default class Scheduling extends Model {
+  @Column({ type: DataType.INTEGER, primaryKey: true })
+  id!: number;
+
+  @Column({ type: DataType.INTEGER })
+  donor_id!: string;
```

Comment:
Can you also leave a comment for now that donor_id would be changed to have a foreign key reference to donors table?

Reply:
Added the comments!